zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -284,13 +285,20 @@ private void checkImportersCompletedAndReschedule() {
                   d -> d instanceof ImportPositionIndex || d instanceof TasklistImportPositionIndex)
               .toList();
 
-      importersCompleted =
-          searchEngineClient.importersCompleted(partitionId, importPositionIndices);
+      final var zeebeIndicesExist =
+          !searchEngineClient
+              .getMappings(
+                  configuration.getIndex().getZeebeIndexPrefix() + "*", MappingSource.INDEX)
+              .isEmpty();
+
+      exporterCanFlush =
+          !zeebeIndicesExist
```

Comment:
🔧 As this is a repeating check, we could also think about whether we want to safe this result, when we the first time encounter zeebe indices, we don't need to recheck for them.